recommender/recommender.py

```diff
@@ -150,8 +203,10 @@ def get_user_is_staff(self):
         being defined. However, It's the only way to get the data right now.
         TODO: Should be proper handled in future
         """
+        if "workbench" in str(type(self.runtime)):
+            return True
```

Comment:
This seems like a bad idea. Why are you trying to specifically detect the workbench?